### DISCUSSION TITLE
Creating container snapshot to be used as mouse cursor

### DISCUSSION THREAD

**Discussion by humbleCoder1990** [pixijs/pixijs#7766] [Q&A]
![Kapture 2021-09-02 at 17 09 31](https://user-images.githubusercontent.com/87828192/131807179-4f1de537-857d-453a-a5f4-74cb75b9b114.gif)

How to make this kind of thing in pixi?

I'm trying to implement mouse drag and drop.
When I start drag, I want to change the cursor's appearance the "mouseDowned" contaienr snapshot.
We can see this kind of feature on many softwares.
I don't want to save the snapshot in computer, just It's enough if it can be in just local variable,
Because it's just instantinated.
My guess is using cacheAsBitmap.
But I can't figure how to reuse it as seperate snapshot like duplicated image..

**Comment by owendwyer** [comment]
You can change the cursor via your renderer

this.renderer.plugins.interaction.cursorStyles.default = 'move';

https://stackoverflow.com/questions/40279317/change-default-cursor-style-in-pixi-js

**Reply by humbleCoder1990** [reply]
I know how to change cursor,
But I want to know how to snapshot my container , then using it as cursor.
